addons/medical_feedback/initSettings.sqf

```diff
@@ -14,3 +14,12 @@
         [true] call FUNC(initEffects);
     }
 ] call CBA_Settings_fnc_init;
+
+[
+    QGVAR(painIntensity),
+    "SLIDER",
+    [LSTRING(PainIntensity_DisplayName), LSTRING(PainIntensity_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory)],
+    [0, 25, 0.6 , 2],
```

Comment:
Are those reasonable limits?

Reply:
At 1 pain I've noticed that pulsing blur starts to look more like a kaleidoscope as the intensity values get higher. For white flashing the maximum is still bearable, so it could be higher. Will update will photo's to illustrate it better when I get the chance.